doc/motion_planning_pipeline/src/motion_planning_pipeline_tutorial.cpp

```diff
@@ -166,9 +168,7 @@ int main(int argc, char** argv)
   // Joint Space Goals
   // ^^^^^^^^^^^^^^^^^
   /* First, set the state in the planning scene to the final state of the last plan */
-  robot_state::RobotState& robot_state = planning_scene->getCurrentStateNonConst();
```

Comment:
I see this that it's fine to delete this line because it's taken care of a few lines later.  :+1:

robot_state.setJointGroupPositions(joint_model_group, response.trajectory.joint_trajectory.points.back().positions);